data-hibernate-reactive/build.gradle

```diff
@@ -9,7 +9,9 @@ dependencies {
     annotationProcessor mn.micronaut.inject.java
     annotationProcessor mn.micronaut.graal
 
-    api projects.micronautDataHibernateJpa
+    api (projects.micronautDataHibernateJpa) {
+        exclude group: 'org.hibernate.orm', module: 'hibernate-core'
```

Comment:
Maybe we should exclude all of the dependencies from the group

Reply:
Don't think we are getting anything besides hibernate-core, but will update it